presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveBucketedTables.java

```diff
@@ -47,7 +47,7 @@
                     "n_comment       STRING) " +
                     "PARTITIONED BY (part_key STRING) " +
                     "CLUSTERED BY (n_regionkey) " +
-                    "INTO 4 BUCKETS " +
+                    "INTO 2 BUCKETS " +
```

Comment:
I'm fine with this change as long as the test remains sound. Does it?

Reply:
According to my best knowledge it remains valid.